src/frontend/types/jwt.ts

```diff
@@ -14,9 +14,10 @@ export interface DecodedJwt {
   roles: string[];
   session_id: string;
   user?: {
+    anonymous_id?: Nullable<string>;
```

Comment:
Are you sure `anonymous_id` is nullable ? You set it in the back only in the case it is existing if I remember.